framework/src/base/FEProblemBase.C

```diff
@@ -129,6 +129,11 @@ validParams<FEProblemBase>()
                         false,
                         "This causes PETSc to error if it had to reallocate memory in the Jacobian "
                         "matrix due to not having enough nonzeros");
+  params.addParam<bool>("ignore_zeros_in_jacobian",
+                        true,
+                        "Do not explicitly store zero values in "
+                        "the Jacobian matrix if "
+                        "ignore_zeros_in_jacobian = true ");
```

Comment:
You can just say "...the Jacobian matrix, if true.", the `ignore_zeros_in_jacobian` is implied by the parameter name ;-)